FrontEnd/src/components/CookiesPolicyPage/LinkContainer.jsx

```diff
@@ -0,0 +1,42 @@
+import { NavLink } from 'react-router-dom';
+import styles from './cookiesPolicyComponent.module.css';
+
+const LinkContainer = () => (
+  <div className={styles['link']}>
+          <NavLink
+            to="/privacy-policy/"
+            className={({ isActive }) =>
+              isActive ? styles['link_text__active'] : styles['link_text']
+            }
+          >
+            Privacy Policy<br />
```

Comment:
It's better to separate links using CSS rather than `<br />`

Reply:
removed the tag br